Admin/RouteAdmin.php

```diff
@@ -109,6 +117,14 @@ public function setContentRoot($contentRoot)
         $this->contentRoot = $contentRoot;
     }
 
+    /**
+     * @deprecated Since 1.4, the resolver should not be used directly, but the RouteDefaults validation.
```

Comment:
this doesn't sound like a complete sentence: "the resolver should not be used directly, but te RouteDefaults validation (...?)"